src/core/lib/iomgr/resource_quota.cc

```diff
@@ -121,26 +123,45 @@ struct grpc_resource_user {
   char* name;
 };
 
-struct grpc_resource_quota {
-  /* refcount */
-  gpr_refcount refs;
+static void rq_step(void* rq, grpc_error* error);
+static void rq_reclamation_done(void* rq, grpc_error* error);
+
+struct grpc_resource_quota : grpc_core::IntrusivelyRefCounted<grpc_resource_quota> {
```

Comment:
All inheritance should be `public`, as per:

https://google.github.io/styleguide/cppguide.html#Inheritance

Reply:
Yes, sure. The base class is now removed.